### PR TITLE
Add groupby.size 

### PR DESCRIPTION
**Pull request by RainFung**
Add groupby.size function by F.count

### CODE REVIEW

**Comment by RainFung** [comment]
This is my first groupby function PR. And i meet some problem when unitest. Can help me fix it?
```
ValueError: Unexpected values: (   B
A   
1  1
2  2
3  3, A
1    1
2    2
3    3
dtype: int64)
```

File: databricks/koalas/tests/test_groupby.py [hunk 1]
```diff
@@ -121,6 +121,13 @@ def test_nunique(self):
         self.assert_eq(kdf.groupby("a").agg({"b": "nunique"}),
                        pdf.groupby("a").agg({"b": "nunique"}))
 
+    def test_size(self):
+        pdf = pd.DataFrame({'A': [1, 2, 2, 3, 3, 3],
+                            'B': [1, 1, 2, 3, 3, 3]})
+        kdf = koalas.DataFrame(pdf)
+
+        self.assert_eq(kdf.groupby("A").size(), pdf.groupby("A").size())
```
Comment (HyukjinKwon) [inline databricks/koalas/tests/test_groupby.py]: pandas returned a series:

```
A
1    1
2    2
3    3
dtype: int64
```

whereas it looks the current API returned a DataFrame.

File: databricks/koalas/groupby.py [hunk 2]
```diff
@@ -344,6 +344,59 @@ def any(self):
...
+        3  3    3
+        Name: count, dtype: int64
+        """
+        groupkeys = self._groupkeys
+        groupkey_cols = [s._scol.alias('__index_level_{}__'.format(i))
+                         for i, s in enumerate(groupkeys)]
+        sdf = self._kdf._sdf
+        data_columns = ['count']
+        sdf = sdf.groupby(*groupkey_cols).count()
+        sdf = sdf.sort(*groupkey_cols)
```
Comment (HyukjinKwon) [inline databricks/koalas/groupby.py]: @RainFung, it's okay not to guarantee the order for now. Let's don't sort if that's the purpose.

File: databricks/koalas/groupby.py [hunk 3]
```diff
@@ -344,6 +344,56 @@ def any(self):
...
+        groupkeys = self._groupkeys
+        groupkey_cols = [s._scol.alias('__index_level_{}__'.format(i))
+                         for i, s in enumerate(groupkeys)]
+        sdf = self._kdf._sdf
+        sdf = sdf.groupby(*groupkey_cols).count()
+        internal = _InternalFrame(sdf=sdf,
+                                  data_columns=['count'],
+                                  index_map=[('__index_level_{}__'.format(i), s.name)
+                                             for i, s in enumerate(groupkeys)])
+        return Series(internal, anchor=DataFrame(internal))
```
Comment (ueshin) [inline databricks/koalas/groupby.py]: The current one causes an error for following operations, e.g.:

```py
>>> kdf.groupby('A').size().sum()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/ueshin/workspace/databricks-koalas/master/databricks/koalas/generic.py", line 908, in sum
    return self._reduce_for_stat_function(F.sum, numeric_only=numeric_only)
  File "/Users/ueshin/workspace/databricks-koalas/master/databricks/koalas/series.py", line 2852, in _reduce_for_stat_function
    col_sdf = sfun(col_sdf)
  File "/Users/ueshin/workspace/databricks-koalas/miniconda/envs/databricks-koalas_3.6/lib/python3.6/site-packages/pyspark/sql/functions.py", line 44, in _
    jc = getattr(sc._jvm.functions, name)(col._jc if isinstance(col, Column) else col)
  File "/Users/ueshin/workspace/databricks-koalas/miniconda/envs/databricks-koalas_3.6/lib/python3.6/site-packages/py4j/java_gateway.py", line 1257, in __call__
    answer, self.gateway_client, self.target_id, self.name)
  File "/Users/ueshin/workspace/databricks-koalas/miniconda/envs/databricks-koalas_3.6/lib/python3.6/site-packages/pyspark/sql/utils.py", line 63, in deco
    return f(*a, **kw)
  File "/Users/ueshin/workspace/databricks-koalas/miniconda/envs/databricks-koalas_3.6/lib/python3.6/site-packages/py4j/protocol.py", line 328, in get_return_value
    format(target_id, ".", name), value)
py4j.protocol.Py4JJavaError: An error occurred while calling z:org.apache.spark.sql.functions.sum.
: java.lang.NullPointerException
	at org.apache.spark.sql.functions$.sum(functions.scala:754)
	at org.apache.spark.sql.functions.sum(functions.scala)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:244)
	at py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:357)
	at py4j.Gateway.invoke(Gateway.java:282)
	at py4j.commands.AbstractCommand.invokeMethod(AbstractCommand.java:132)
	at py4j.commands.CallCommand.execute(CallCommand.java:79)
	at py4j.GatewayConnection.run(GatewayConnection.java:238)
	at java.lang.Thread.run(Thread.java:748)
```

We should use `Series._col`.

```py
return _col(DataFrame(internal))
```

**Comment by ueshin** [comment]
Thanks! merging.

**Pull request merged**